packages/bridge/smart-contract-templates/icon/icon_hello/icon_hello.py.hbs

```diff
@@ -32,7 +32,7 @@ class hello(IconScoreBase):
     def cre{{name}}(self{{#each fields}}, {{name}}: {{type}}{{/each}}):
         inputData = {}
         {{#each fields}}
-        inputData["{{name}}"] = {{name}}
+        inputData["{{name}}"] = str({{name}})
```

Comment:
Is it optimized by always using `str()`?

Reply:
Some data type is not possible to Json stringify such as Address. so that I convert it to string type first.

Reply:
continue with #72